docs/api/index.md

```diff
@@ -14,176 +14,175 @@ For compatibility with Jest, `TestFunction` can also be of type `(done: DoneCall
 
 ## test
 
-`test` defines a set of related expectations. Default timeout for tests is 5 seconds, and can be configured globally with [testTimeout](../config/#testtimeout).
+- **Type:** `(name: string, fn: TestFunction, timeout?: number) => void`
+- **Alias:** `it`
 
-**Type:** `(name: string, fn: TestFunction, timeout?: number) => void`
-**Alias:** `it`
+  `test` defines a set of related expectations. Default timeout for tests is 5 seconds, and can be configured globally with [testTimeout](../config/#testtimeout).
 
-```ts
-import { test, expect } from 'vitest'
+  ```ts
+  import { test, expect } from 'vitest'
 
-test('should work as expected', () => {
-  expect(Math.sqrt(4)).toBe(2);
-})
-```
+  test('should work as expected', () => {
+    expect(Math.sqrt(4)).toBe(2);
+  })
+  ```
```

Comment:
Some strange happened with formatting here

Reply:
Nvm, ignore

Reply:
what do you mean? this is how vite code docs are formatted: https://raw.githubusercontent.com/vitejs/vite/main/docs/config/index.md

Reply:
I missread the diff, thus the nvm, ignore the noise 😁

Reply:
Yeah. 😂